src/main/java/ro/msg/learning/shop/repository/OrderDetailRepository.java

```diff
@@ -0,0 +1,11 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.entity.OrderDetailEntity;
+
+import java.util.UUID;
+
+@Repository
+public interface OrderDetailRepository extends JpaRepository<OrderDetailEntity, UUID>
```

Comment:
Should be public interface OrderDetailRepository extends JpaRepository<OrderDetailEntity, OrderDetailId>  (OrderDetailId is the type of the id in the OrderDetailEntity)